src/plays/calendar/Context.tsx

```diff
@@ -0,0 +1,140 @@
+import { format } from "date-fns";
+import React, { useState, useCallback } from "react";
+import { orderBy } from 'lodash'
+
+export const Context = React.createContext<any>({
+  modalTitle: '',
+  modalContent: undefined,
+  getEvents: (date: Date) => {},
+  addEvent: (event: any) => {},
+  updateEvent: (event: any) => {},
+  deleteEvent: (event: any) => {},
+  showModal: (content: React.ReactNode, title?: '') => {},
+  hideModal: () => {}
+})
+
+export const ContextProvider = ({ children }: any) => {
+  const [events, setEvents] = useState<any[]>([
+    {
+      id: '1',
```

Comment:
It will be great to distribute the default fake events spread across multiple days. It will look more catchy when someone opens it up for the first time!